### PR TITLE
compose tunnel agent esbuild

### PR DESCRIPTION
**Pull request by royra**
- compose-tunnel-agent: esbuild

### CODE REVIEW

File: packages/cli/src/lib/docker-proxy-client.ts [hunk 1]
```diff
@@ -12,7 +12,6 @@ const DOCKER_PROXY_PORT = 3000
 const baseDockerProxyService: ComposeService = {
   build: {
     context: DOCKER_PROXY_DIR,
```
Comment (Yshayy) [inline packages/cli/src/lib/docker-proxy-client.ts:14]: We might need to add `target: 'production'`

File: packages/compose-tunnel-agent/build.mjs [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import * as esbuild from 'esbuild'
+
+await esbuild.build({
+  entryPoints: ['main.ts', 'index.ts'],
```
Comment (Yshayy) [inline packages/compose-tunnel-agent/build.mjs:4]: What does it mean that we have two entry points?

**Review by Yshayy — COMMENTED** [review]
Looks good, have you verified it works in a clean environment? (production setup, out of working dir/workspace node_modules, standalone cli)

File: packages/cli/src/lib/docker-proxy-client.ts (hunk 1 above)
Comment (royra) [reply packages/cli/src/lib/docker-proxy-client.ts:14]: the default is the last builder

**Pull request merged**